website/docs/10_going-deeper/environment-configuration.mdx

```diff
@@ -42,4 +42,15 @@ This way, you can have different configurations depending on your needs.
 
 Booster environments are extremely flexible. As shown in the first example, your 'fruit-store' app can have three team-wide environments: 'dev', 'stage', and 'prod', each of them with different app names or providers, that are deployed by your CI/CD processes. Developers, like "John" in the second example, can create their own private environments in separate config files to test their changes in realistic environments before committing them. Likewise, CI/CD processes could generate separate production-like environments to test different branches to perform QA in separate environments without interferences from other features under test.
 
-The only thing you need to do to deploy a whole new completely-independent copy of your application is to use a different name. Also, Booster uses the credentials available in the machine (`~/.aws/credentials` in AWS) that performs the deployment process, so developers can even work on separate accounts than production or staging environments.
\ No newline at end of file
+The only thing you need to do to deploy a whole new completely-independent copy of your application is to use a different name. Also, Booster uses the credentials available in the machine (`~/.aws/credentials` in AWS) that performs the deployment process, so developers can even work on separate accounts than production or staging environments.
+
+## Advanced Configuration Management
+
+For managing dynamic configuration values that can be updated without redeployment, Booster provides a comprehensive configuration management system with support for external configuration providers like Azure App Configuration.
+
+See the [Configuration Management](/features/configuration) documentation for detailed information about:
```

Comment:
Same as below. I'm not sure if this link to `/features/configuration` is going to link to an existing page. The documentation section created above has this path `website/docs/03_features/03_configuration.mdx`. Could you double check that the documentation pages are properly linked?